spark/v3.2/spark/src/main/java/org/apache/iceberg/spark/procedures/RemoveOrphanFilesProcedure.java

```diff
@@ -51,7 +55,10 @@ public class RemoveOrphanFilesProcedure extends BaseProcedure {
       ProcedureParameter.optional("location", DataTypes.StringType),
       ProcedureParameter.optional("dry_run", DataTypes.BooleanType),
       ProcedureParameter.optional("max_concurrent_deletes", DataTypes.IntegerType),
-      ProcedureParameter.optional("file_list_view", DataTypes.StringType)
+      ProcedureParameter.optional("file_list_view", DataTypes.StringType),
+      ProcedureParameter.optional("equal_schemes", STRING_MAP),
+      ProcedureParameter.optional("equal_authorities", STRING_MAP),
+      ProcedureParameter.optional("prefix_mismatch_mode", STRING_MAP),
```

Comment:
I don't think it should be `STRING_MAP`. I think it should be `DataTypes.StringType`. Can we also add tests to make sure this is tested?

Reply:
added.